src/simulation/dynamics/Integrators/_Documentation/secTest.tex

```diff
@@ -63,9 +67,9 @@ \subsection{Test sections}
 The resulting data points are illustrated in Figure~\ref{fig:intResults}.  The large 120 second time step causes all the integration methods to significantly deviate from the truth locations (shown in black).  The integrated validation test ensures that the BSK integrations yield the same integration corruptions to validate the mathematical implementation.
 
 \subsection{Test success criteria}
-The integrated position states are checked at 5 even data points along the simulation time again pre-computed truth answers.  These truth answers were generated in Mathematica implementing the same initial conditions, and replicating the integration math.    The accuracy threshold is set to 1 meter, a small value compared to the 7,000,000 meter near-circular orbit radius.
-
+For the fixed-step integrators, the integrated position states are checked at 5 even data points along the simulation time againt pre-computed truth answers.  These truth answers were generated in Mathematica implementing the same initial conditions, and replicating the integration math.    The accuracy threshold is set to 1 meter, a small value compared to the 7,000,000 meter near-circular orbit radius.
```

Comment:
Regarding doing new integrator tests, the nice thing about the current setup is that the tests run very fast, and we are checking the values for a handful of steps to ensure each value is what we expect. If we doing do spot checks are apoapses and periapses, we will have to integrated the states of a longer time period to which can drastically increase the time to run these tests. I'm not seeing any benefit to running such slower test beyond that the truth values are computed on the fly rather than crafted offline. I would like to stay with our current testing method for now.